spec/mongoid/query_cache_spec.rb

```diff
@@ -611,7 +624,7 @@
       Mongoid::QueryCache.enabled = true
       10.times { Band.create! }
 
-      Band.batch_size(4).all.any?
+      Band.batch_size(4).to_a
```

Comment:
I found out that calling `any?` with no block was actually causing a session leak. I documented it in [RUBY-2391](https://jira.mongodb.org/browse/RUBY-2391).

Reply:
Does this leak apply to Mongoid's QC code also?

Reply:
Good question. I'll make a separate PR after this one.